### PR TITLE
Changed arriving to into "Travelling to Pokestop: {0} ({1}m Away)"), and fixed console output need 1 change to rocket-api

### PR DESCRIPTION
**Pull request by DaveG9856**
AS requested Changed arriving to into "Travelling to Pokestop: {0} ({1}m) Away"),

### CODE REVIEW

**Comment by DaveG9856** [comment]
fixed

**Comment by XeonPowder** [comment]
:D dats the way uhuh uhuh i like it.

**Comment by DaveG9856** [comment]
lol

File: PoGo.PokeMobBot.Logic/Common/Translations.cs [hunk 1]
```diff
@@ -186,7 +186,7 @@ public class Translation : ITranslation
             new KeyValuePair<TranslationString, string>(TranslationString.EventFortFailed,
                 "Name: {0} INFO: Looting failed, possible softban. Unban in: {1}/{2}"),
             new KeyValuePair<TranslationString, string>(TranslationString.EventFortTargeted,
-                "Arriving to Pokestop: {0} in ({1}m)"),
+                "Travelling to Pokestop: {0} ({1}m) Away"),
```
Comment (DurtyFree) [inline PoGo.PokeMobBot.Logic/Common/Translations.cs]: Should better be "Travelling to Pokestop: {0} ({1}m Away)"

**Comment by DurtyFree** [comment]
@DaveG9856 Lets do this properly pls :D See my comment

**Comment by DaveG9856** [comment]
fixed, noticed that myself lol

**Comment by DaveG9856** [comment]
Second part of this updates console ouput
Fixed
[22:09:09] (POKESTOP) Name:  blah XP: 50, Gems: 0, Items: 1 x GreatBall, 2 x PokeBall
as opposed to
[22:07:13] (POKESTOP) Name: blah  XP: 50, Gems: 0, Items: 3 x ItemPokeBall

Needs 1 fix done to 
Pokemon-Go-Rocket-API/PokemonGo.RocketAPI/Rpc/Inventory.cs   line 70
ItemId = ItemId.ItemLuckyEgg to ItemId = ItemId.LuckyEgg
that's it

**Comment by DurtyFree** [comment]
You should not touch the Protos nor the RocketAPI. :/

Please recommit with only the text changes.

**Pull request closed without merging**